docs/contribute.md

```diff
@@ -0,0 +1,157 @@
+# Contributing Guide
+
+Thank you for deciding to contribute and help us improve Percona Everest documentation!
+
+We welcome contributors from all users and community. By contributing, you agree to the [Percona Community code of conduct](https://percona.community/contribute/coc/).
+
+You can contribute to documentation in the following ways:
+
+1. **Request a doc change through a Jira issue**. If you’ve spotted a doc issue (a typo, broken links, inaccurate instructions, etc.) but don’t have time nor desire to fix it yourself - let us know about it.
+
+	- Click the **Jira** link in the contact us section. This opens the [Jira issue tracker](https://jira.percona.com/projects/Everest).
+	- Sign in (create a Jira account if you don’t have one) and click **Create** to create an issue.
+	- Describe the issue you have detected in the Summary, Description, Steps To Reproduce, Affects Version fields.
+
+2. **[Contribute to documentation yourself](#contribute-to-documentation-yourself)**. There is the **Edit this page** link that leads you to the source file of the page on GitHub. There you make changes, create a pull request that we review and add to the doc project. For details how to do it, read on.
+
+## Contribute to documentation yourself
+
+To contribute to the documentation, you should be familiar with the following technologies:
+- [Markdown] markup language. We write the documentation in it.
+- [MkDocs] generator. We use it to convert source ``.md`` files to html and PDF documents.
+- [git](https://git-scm.com/) and [GitHub](https://guides.github.com/activities/hello-world/)
+- [Docker]. It allows you to run MkDocs in a virtual environment instead of installing it and its dependencies on your machine.
+
+The `.md` files are in the ``docs/`` directory. 
```

Comment:
With regard to the double backticks: are they necessary? I don't know how to review a PR, where there are both single and double backticks on the same line :) What is going to happen if we ensure a consistent use of single backticks?

Reply:
ok